### PR TITLE
Adds an error boundary to avoid showing a blank page

### PR DESCRIPTION
**Pull request by madsnedergaard**
### Description

This catches errors and shows them in a nicer way to users (instead of a blank page), while allowing users to return to the map.

#### Preview

Screenshot 2022-01-11 at 10 04 43

### CODE REVIEW

**Review by Kongkille — APPROVED** [review]
This is a great idea!

File: web/src/components/errorboundary.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    // TODO: Send this to Sentry
+    console.error(error, errorInfo);
+  }
+
+  render() {
+    if (this.state.hasError) {
+      const url = window.location.href;
+      return (
+        <ErrorBox>
+          <h1>Oh no, something went terribly wrong...</h1>
```
Comment (Kongkille) [inline web/src/components/errorboundary.js:57]: Perhaps just "Oh no, something went wrong ..." so it comes across less catastrophic

File: web/src/components/errorboundary.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          <p>
+            Please let us know{' '}
+            <a href="https://github.com/tmrowco/electricitymap-contrib/">on Github</a> so we can fix
```
Comment (skovhus) [inline web/src/components/errorboundary.js:60]:
```suggestion
            <a href="https://github.com/electricitymap/electricitymap-contrib">on Github</a> so we can fix
```

File: web/src/components/errorboundary.js (hunk 1 above)
Comment (skovhus) [reply web/src/components/errorboundary.js:57]:
```suggestion
          <h1>Oh no, something went wrong...</h1>
```

**Pull request merged**